src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishReadyToRun.cs

```diff
@@ -236,8 +236,11 @@ public void It_supports_libraries_when_using_crossgen2(string targetFramework)
 
             var testProjectInstance = _testAssetsManager.CreateTestProject(testProject);
 
-            var publishCommand = new PublishCommand(Log, Path.Combine(testProjectInstance.Path, testProject.Name));
-            publishCommand.Execute().Should().Pass();
+            var publishCommand = new PublishCommand(testProjectInstance);
+            publishCommand.Execute()
+                .Should()
+                .Fail()
+                .And.HaveStdOutContainingIgnoreCase("NETSDK1126");
```

Comment:
@dsplaisted was this an intentional change and should it flow into master? It looks like this test is failing.

Reply:
Looks like a bad merge.  The test was renamed and changed so that it should pass in the master branch: https://github.com/dotnet/sdk/pull/14678